### PR TITLE
Create Page For Logging Users In

### PR DESCRIPTION
**Pull request by mahadzaryab1**
### Notion ticket link

[Frontend: Create login](https://www.notion.so/uwblueprintexecs/Frontend-Create-login-838f1bd608c84748964b623f49ad72a0)

### Implementation description

- Created a login page that users will use to sign in or register to the platform

### Steps to test

1. Run the app locally and head over to http://localhost:3000/login to see the new login page

### Checklist

-   [x] My PR name is descriptive and in imperative tense
-   [x] I have run the linter
-   [x] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: pages/_app.tsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import "@fontsource/roboto/400.css";
```
Comment (feng-eric) [inline pages/_app.tsx:1]: Should we import all font stuff to a file and export it out from there so we standardize the font we use? I feel like people won't know what font to use and we'll have diff fonts throughout the app

**Review by feng-eric — COMMENTED** [review]
If you want it deployed for others to see, you have to join the railway project with the github acc you're using for blueprint

File: pages/_app.tsx (hunk 1 above)
Comment (mahadzaryab1) [reply pages/_app.tsx:1]: the point of this file is to standardize the font to use - the font(s) defined here will be used by default

**Review by feng-eric — APPROVED** [review]
LGTM!

**Review by victoranyun — APPROVED** [review]

**Pull request merged**